checkov/common/bridgecrew/integration_features/features/suppressions_integration.py

```diff
@@ -106,17 +141,22 @@ def _apply_suppressions_to_report(self, scan_report: Report) -> None:
         scan_report.failed_checks = still_failed_checks
         scan_report.passed_checks = still_passed_checks
 
-    def _check_suppressions(self, record: Record, suppressions: list[dict[str, Any]]) -> dict[str, Any] | None:
+    def _check_suppressions(self, record: Record, suppressions: Optional[list[dict[str, Any]]], suppressions_v2: Optional[list[dict[str, Any]]]) -> dict[str, Any] | None:
         """
-        Checks the specified suppressions against the specified record, returning the first applicable suppression,
-        or None of no suppression is applicable.
+        Checks the specified suppressions against the specified record, returning a tuple of the applied suppression and whether
+        the suppression was a v1 suppression or not. If no suppression is found, then returns a tuple of None, None
```

Comment:
by the function type hints it looks like that the function returns a list of dictionaries, not tuples. what should it be?

Reply:
Good catch, this was outdated from an earlier version of the method I decided to change